### PR TITLE
Fix RPATH problem

### LINKED ISSUE #25: Linux pkg-config files are unusable
The pkg-config files for the brotli Linux builds contain an option `-R` that is not supported by x86_64-conda-linux-gnu-cc

I think this was introduced during the last version bump. Any chance that this is related to the patch that is currently applied?

I noticed this when bumping [libsoup](https://github.com/conda-forge/libsoup-feedstock/pull/4):

```
[91/103] $BUILD_PREFIX/bin/x86_64-conda-linux-gnu-cc  -o libsoup/libsoup-2.4.so.1.11.0 libsoup/libsoup-2.4.so.1.11.0.p/meson-generated_.._soup-resources.c.o libsoup/libsoup-2.4.so.1.11.0.p/meson-generated_.._soup-enum-types.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-address.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-basic.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-digest.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-ntlm.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-negotiate.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-domain.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-domain-basic.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-domain-digest.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-manager.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-body-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-body-output-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cache.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cache-client-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cache-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-client-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-connection.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-connection-auth.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-decoder.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-processor.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-sniffer.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-sniffer-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-converter-wrapper.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie-jar.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie-jar-db.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie-jar-text.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-date.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-directory-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-filter-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-form.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-hsts-enforcer.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-hsts-enforcer-db.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-hsts-policy.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-headers.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-init.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-io-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-logger.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-body.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-client-io.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-headers.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-io.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-queue.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-server-io.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-method.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-misc.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-multipart.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-multipart-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-password-manager.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-path-map.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-resolver.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-resolver-default.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-resolver-wrapper.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-uri-resolver.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request-data.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request-file.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request-http.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-requester.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-server.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session-async.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session-feature.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session-sync.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-socket.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-socket-properties.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-status.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-tld.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-uri.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-value-utils.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-version.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-connection.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-extension.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-extension-deflate.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-extension-manager.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-xmlrpc.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-xmlrpc-old.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-brotli-decompressor.c.o -L$PREFIX/lib -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libsoup-2.4.so.1 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/libsoup-2.72.0 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib $PREFIX/lib/libglib-2.0.so $PREFIX/lib/libgobject-2.0.so $PREFIX/lib/libgio-2.0.so $PREFIX/lib/libxml2.so $PREFIX/lib/libsqlite3.so $PREFIX/lib/libpsl.so -R$PREFIX/lib $PREFIX/lib/libbrotlidec.so $PREFIX/lib/libz.so -Wl,--end-group
FAILED: libsoup/libsoup-2.4.so.1.11.0 
$BUILD_PREFIX/bin/x86_64-conda-linux-gnu-cc  -o libsoup/libsoup-2.4.so.1.11.0 libsoup/libsoup-2.4.so.1.11.0.p/meson-generated_.._soup-resources.c.o libsoup/libsoup-2.4.so.1.11.0.p/meson-generated_.._soup-enum-types.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-address.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-basic.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-digest.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-ntlm.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-negotiate.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-domain.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-domain-basic.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-domain-digest.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-auth-manager.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-body-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-body-output-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cache.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cache-client-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cache-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-client-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-connection.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-connection-auth.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-decoder.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-processor.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-sniffer.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-content-sniffer-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-converter-wrapper.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie-jar.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie-jar-db.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-cookie-jar-text.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-date.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-directory-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-filter-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-form.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-hsts-enforcer.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-hsts-enforcer-db.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-hsts-policy.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-headers.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-init.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-io-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-logger.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-body.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-client-io.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-headers.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-io.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-queue.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-message-server-io.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-method.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-misc.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-multipart.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-multipart-input-stream.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-password-manager.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-path-map.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-resolver.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-resolver-default.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-resolver-wrapper.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-proxy-uri-resolver.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request-data.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request-file.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-request-http.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-requester.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-server.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session-async.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session-feature.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-session-sync.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-socket.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-socket-properties.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-status.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-tld.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-uri.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-value-utils.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-version.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-connection.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-extension.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-extension-deflate.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-websocket-extension-manager.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-xmlrpc.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-xmlrpc-old.c.o libsoup/libsoup-2.4.so.1.11.0.p/soup-brotli-decompressor.c.o -L$PREFIX/lib -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libsoup-2.4.so.1 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/libsoup-2.72.0 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib $PREFIX/lib/libglib-2.0.so $PREFIX/lib/libgobject-2.0.so $PREFIX/lib/libgio-2.0.so $PREFIX/lib/libxml2.so $PREFIX/lib/libsqlite3.so $PREFIX/lib/libpsl.so -R$PREFIX/lib $PREFIX/lib/libbrotlidec.so $PREFIX/lib/libz.so -Wl,--end-group
x86_64-conda-linux-gnu-cc: error: unrecognized command line option '-R'
```

### PR DESCRIPTION
**Pull request by tschoonj**
Closes #25

Looks like it was reported and fixed upstream: https://github.com/google/brotli/issues/836

### CODE REVIEW

**Comment by tschoonj** [comment]
Could someone merge this one in please? It's blocking updates to the libsoup feedstock 

Thanks!

**Comment by tschoonj** [comment]
@xhochy  @isuruf  @beckermr @marcelotrevisani Can someone merge this in please?

Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,10 @@ source:
   sha256: f9e8d81d0405ba66d181529af42a3354f838c939095ff99930da6aa9cdf6fe46
   patches:
     - 0001-Add-separate-options-to-disable-shared-static-librar.patch
+    - 0002-Fix-pkgconfig-rpath.patch
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:14]: Could you please add a comment here explaining that after the release ``1.0.9`` this patch will be no longer necessary?

Comment (tschoonj) [reply recipe/meta.yaml:14]: Done!

Comment (marcelotrevisani) [reply recipe/meta.yaml:14]: Thanks! :)

**Review by marcelotrevisani — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (beckermr) [reply recipe/meta.yaml:14]: This is great. In the future, you can use jinja2 if statements to turn patches off if the version is not right. This helps automate things.

**Comment by tschoonj** [comment]
Thanks!

**Pull request merged**